CMakeLists.txt

```diff
@@ -168,6 +175,28 @@ else()
   message(FATAL_ERROR "ufs-bundle unknown UFS_APP ${UFS_APP}")
 endif()
 
+
+# ioda, ufo and fv3-jedi test data
+#----------------------------------------
+
+# When LOCAL_PATH_JEDI_TESTFILES is set to the directory of IODA test files stored
+# in a local directory, ioda-data repo will not be cloned
+if( NOT DEFINED ENV{LOCAL_PATH_JEDI_TESTFILES} )
```

Comment:
Note that this is not how we want to handle those switches in the future. Instead, the logic in the bundles should be like here: https://github.com/JCSDA-internal/jedi-bundle/blob/551edcbc73700544994c2b10fa1270fa25c1900a/CMakeLists.txt#L51C1-L51C91